### DISCUSSION TITLE
add ByteBuf monitor

### DISCUSSION THREAD

**Discussion by 342963711** [netty/netty#13890] [Q&A] [answered]
Under the default configuration, the direct memory used by netty cannot be observed through MBeans. Hope to add ByteBuf's monitor
<img width="296" alt="image" src="https://github.com/netty/netty/assets/22129678/6a3f9bd6-4194-420b-99a5-3f8facfe85e3">

**Comment by franz1981** [comment, marked as answer]
If you set `-Dio.netty.maxDirectMemory=0` or disable `Unsafe`, `ByteBuf`s should use `Cleaner`s and should be visible as any other direct memory user. Without, you need to use the allocator statistics or, if you're using an higher-level framework using Netty, it is often possible to read such metrics with the right micrometer plugin.
